lib/hello.js

```diff
@@ -5,7 +5,11 @@ var cors = require('cors');
 function helloRoute() {
   var hello = new express.Router();
   hello.use(cors());
-  hello.use(bodyParser());
+  //hello.use(bodyParser());
```

Comment:
Cool, I'll leave as it is and just remove the commented out code as the deprecated warnings are gone